src/nupic/os/Timer.cpp

```diff
@@ -66,38 +66,6 @@ static UInt64 getCurrentTime()
   return (UInt64)(v.QuadPart) - initialTicks_;
 }
 
-#elif defined(NTA_PLATFORM_darwin)
```

Comment:
I think it is highly likely that `NTA_PLATFORM_darwin` used to be set correctly but accidentally stopped getting set due to build changes. As such, we need to decide whether this code is the right way to do timing on OS X or not.

The fact that there is a bug in the code is not sufficient reason to throw it away. In fact, it may be a bug that impacts other areas as well so we should figure it out before we decide whether to remove this code.

@subutai - Do you have any background on using high precision timers on OS X like this code does? Do we need high precision like this? Did we intentionally stop using this code block?